packages/freetype/project.bri

```diff
@@ -33,28 +33,28 @@ export default function freetype(): std.Recipe<std.Directory> {
     .workDir(source)
     .dependencies(std.toolchain, brotli, libpng)
     .toDirectory()
-    .pipe(
-      (source) =>
-        // Fix absolute paths in the script freetype-config
-        std.runBash`
+    .pipe((source) =>
+      // Fix absolute paths in the script freetype-config
+      std.runBash`
           sed -i 's|  prefix=".*"|  prefix=\`dirname "\$0"\`/..|' "$BRIOCHE_OUTPUT/bin/freetype-config"
           sed -i 's|  exec_prefix=".*"|  exec_prefix=\${prefix}|' "$BRIOCHE_OUTPUT/bin/freetype-config"
           sed -i 's|  libdir=".*"|  libdir=\${prefix}/lib|' "$BRIOCHE_OUTPUT/bin/freetype-config"
           sed -i 's|  includedir=".*"|  includedir=\${prefix}/include|' "$BRIOCHE_OUTPUT/bin/freetype-config"
         `
```

Comment:
Small indentation fix (just opened https://github.com/brioche-dev/brioche/issues/354 since it'd be beter if `brioche fmt` handled this automatically)

```suggestion
      std.runBash`
        sed -i 's|  prefix=".*"|  prefix=\`dirname "\$0"\`/..|' "$BRIOCHE_OUTPUT/bin/freetype-config"
        sed -i 's|  exec_prefix=".*"|  exec_prefix=\${prefix}|' "$BRIOCHE_OUTPUT/bin/freetype-config"
        sed -i 's|  libdir=".*"|  libdir=\${prefix}/lib|' "$BRIOCHE_OUTPUT/bin/freetype-config"
        sed -i 's|  includedir=".*"|  includedir=\${prefix}/include|' "$BRIOCHE_OUTPUT/bin/freetype-config"
      `
```